plugins/jade/src/main/scala/org/broadinstitute/monster/sbt/ClassGenerator.scala

```diff
@@ -1,13 +1,12 @@
 package org.broadinstitute.monster.sbt
 
+import java.nio.file.{Files, Path}
```

Comment:
The changes in this file are from moving code out of the plugin class itself (it was getting pretty gnarly)